pkg/quarterdeck/db/migrations/0002_default_data.sql

```diff
@@ -9,29 +9,31 @@ INSERT INTO roles (id, name, description, created, modified) VALUES
-    (17, 'subscriber', 'Is allowed to subscribe to topics in the project to consume events', true, false, datetime('now'), datetime('now'))
+    (1, 'organizations:create', 'Can create new organizations', false, true, datetime('now'), datetime('now')),
+    (2, 'organizations:delete', 'Can delete organizations and their data', false, true, datetime('now'), datetime('now')),
+    (3, 'organizations:list', 'Can view the list of organizations', false, true, datetime('now'), datetime('now')),
+    (4, 'organizations:edit', 'Can make changes to organizations', false, true, datetime('now'), datetime('now')),
+    (5, 'organizations:detail', 'Can view details of an organization', false, true, datetime('now'), datetime('now')),
+    (6, 'collaborators:add', 'Can add users to an organization and change their roles', false, true, datetime('now'), datetime('now')),
+    (7, 'collaborators:remove', 'Can remove users from an organization', false, true, datetime('now'), datetime('now')),
+    (8, 'collaborators:edit', 'Can modify existing users in an organization', false, true, datetime('now'), datetime('now')),
+    (9, 'collaborators:read', 'Can see the collaborators for an organization', false, true, datetime('now'), datetime('now')),
+    (10, 'projects:edit', 'Can create and update projects in the organization', false, true, datetime('now'), datetime('now')),
+    (11, 'projects:delete', 'Can delete projects from the organization along with their data', false, true, datetime('now'), datetime('now')),
+    (12, 'projects:read', 'Can view projects in the organization', false, true, datetime('now'), datetime('now')),
+    (13, 'apikeys:edit', 'Can create and update api keys in a project', false, true, datetime('now'), datetime('now')),
+    (14, 'apikeys:delete', 'Can delete api keys from a project', false, true, datetime('now'), datetime('now')),
+    (15, 'apikeys:read', 'Can view api keys inside of a project', false, true, datetime('now'), datetime('now')),
+    (16, 'topics:create', 'Can create topics in a project', true, true, datetime('now'), datetime('now')),
+    (17, 'topics:edit', 'Can modify topic metadata and mark as read only', true, true, datetime('now'), datetime('now')),
+    (18, 'topics:destroy', 'Can destroy topics and all their data', true, true, datetime('now'), datetime('now')),
+    (19, 'topics:read', 'Can view the topics for a project', true, true, datetime('now'), datetime('now')),
+    (20, 'metrics:read', 'Can view the metrics of topics in the specified project', true, true, datetime('now'), datetime('now')),
+    (21, 'publisher', 'Is allowed to publish events to the topics in the project', true, false, datetime('now'), datetime('now')),
+    (22, 'subscriber', 'Is allowed to subscribe to topics in the project to consume events', true, false, datetime('now'), datetime('now'))
 ;
 
 INSERT INTO role_permissions (role_id, permission_id, created, modified) VALUES
-    (1, 1, datetime('now'), datetime('now')),
```

Comment:
I'm going to create a test that checks the permissions and roles to make it easier to edit this in the future. I'll commit it to this branch so it joins this PR - give me a couple of minutes.

Reply:
This is done!